wiki/Beatmap/Pattern/Mania/Long Notes/en.md

```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - jacks
+---
+
+# Long Notes
+
+Also referred to in-game as holds, **long notes** are notes that require you to hold and release them afterwards. They are often referred to as LNs for short and are used to represent prolonged sounds in the music.
```

Comment:
Maybe add that some people call them noodles, similar with rice on notes.

Reply:
```suggestion
Also referred to as holds in the editor, **long notes** are notes that require you to hold and release them afterwards. They are often referred to as noodles or LNs and are used to represent prolonged sounds in the music.
```